Userland/Utilities/fortune.cpp

```diff
@@ -74,7 +74,7 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
 {
     TRY(Core::System::pledge("stdio rpath"));
 
-    StringView path = "/res/fortunes.json"sv;
```

Comment:
Doing this will also require an update to the [Discord bot](https://github.com/SerenityOS/discord-bot/blob/649643dd08efa2ee03f7b31ff7244da2ad8d6bcd/src/apis/githubAPI.ts#L62).